static/customWidgets/GeometryWidget/geometrywidget.js

```diff
@@ -35,6 +34,35 @@
       }),
     });
 
+    ;
+
+    var layers =  this.options.wms_layers.split(',');
+    var ol_layers = [];
+    for (var i=0; i < layers.length; i++) {
+        ol_layers.push(
+            new ol.layer.Image({
+            source: new ol.source.ImageWMS({
+              url: layers[i],
+              projection: "EPSG:2056",
```

Comment:
What happens if a layer in the groups comes with an other projection system? Maybe it's more robust to parse and extract the CRS parameter from the WMS URL?